tests/test_cli.py

```diff
@@ -149,6 +94,72 @@ def test_model_upload(mocker, tmp_path):
     assert args[3] == ["torch==1.13.1", "pandas<=1.5.0"]
 
 
+@pytest.mark.cli
```

Comment:
🎊🤩 Hooray! These tests look 💯 fantastic! 💯 Love to learn 🧠✨ that this parameterized tests feature exists, hadn't really seen that before! 🤯🥳 You're rockin' it! 🎸🌟

Reply:
Yeah they rock! For some more background on the practice, this is a good primer from the Go world: https://dave.cheney.net/2019/05/07/prefer-table-driven-tests